install/helmcharts/meshery/templates/clusterrole.yaml

```diff
@@ -5,10 +5,12 @@ metadata:
   labels:
 {{- include "meshery.labels" . | nindent 4 }}
 rules:
-- apiGroups:
-  - '*'
+- apiGroups: ["networking.istio.io"]
```

Comment:
please revert this back to `*` as meshery operator needs this.

Reply:
@kumarabd can we get more specific, please? Will you create a page on docs.meshery.io with the required permissions?